packages/mdc-textfield/_mixins.scss

```diff
@@ -406,6 +406,50 @@
   }
 }
 
+// Icons
+
+@mixin mdc-text-field-with-leading-icon_ {
+  .mdc-text-field__input {
+    @include mdc-rtl-reflexive-property(padding, $mdc-text-field-icon-padding, $mdc-text-field-icon-position);
+  }
+
+  .mdc-text-field__icon {
+    @include mdc-rtl-reflexive-position(left, $mdc-text-field-icon-position);
+  }
+
+  .mdc-text-field__label {
+    @include mdc-rtl-reflexive-position(left, $mdc-text-field-icon-padding);
+  }
+}
+
+@mixin mdc-text-field-outlined-with-leading-icon_ {
```

Comment:
Do you want to group these with the outlined mixins?

Reply:
I dont think so. I think I want to cover all cases of icons (leading or trailing) in the same code grouping. Lots of the styling seems to revolve more styling the icon.